src/dp_periodic_msg.c

```diff
@@ -30,6 +30,12 @@ void send_to_all_vfs(const struct rte_mbuf *pkt, uint16_t eth_type)
 		if (port->is_pf || !port->allocated)
 			continue;
 
+		if (eth_type == RTE_ETHER_TYPE_IPV6 && !port->iface.is_ipv6_set)
+			continue;
+
+		if (eth_type == RTE_ETHER_TYPE_IPV4 && port->iface.cfg.own_ip == 0)
+			continue;
```

Comment:
May not be a real problem, but what about other ETHER_TYPEs? Maybe an inline `dp_is_ip_set(port)` or something that does `return (eth_type == RTE_ETHER_TYPE_IPV6 && port->iface.is_ipv6_set) || eth_type == RTE_ETHER_TYPE_IPV4 && port->iface.cfg.own_ip != 0);` ?

Reply:
Good catch. Actually ARP needs also consideration here.